website/core/Footer.js

```diff
@@ -41,7 +41,7 @@ class Footer extends React.Component {
           <div>
             <h5>Community</h5>
             <a
-              href={this.props.config.baseUrl + this.props.language + "/users/"}
```

Comment:
I think the proper fix should be to make `this.props.language` defaults to `en` and so does `About` and `Usage`.